firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalSerializerTest.java

```diff
@@ -404,6 +404,51 @@ public void testEncodesTargetData() {
     assertEquals(targetData, decoded);
   }
 
+  @Test
+  public void localSerializerDropsExpectedCountInTargetData() {
```

Comment:
nit: Please rename this test method slightly to "localSerializerShouldDropExpectedCountInTargetData". This more clearly articulates what the _expected_ behavior is.